_targets.R

```diff
@@ -0,0 +1,36 @@
+library(targets)
+source("code.R")
```

Comment:
It's generally preferable to source the individual src folder functions rather than combine all of them into one script. The instructions make it sound like making `code.R` is preferred (I filed an issue about this [here](https://github.com/USGS-R/ds-pipelines-targets-1-course/issues/17))

Reply:
I don't think it matters for this training, though.